packages/hardhat/abi/@aragon/apps-finance/contracts/Finance.sol/Finance.json

```diff
@@ -0,0 +1,1009 @@
+[
```

Comment:
why we need apps abis that are not going to be installed on the gardens template?

Reply:
Yes, you were right, there was a lot of ABIs not needed. I revert that change on #81